### ISSUE TITLE
header element blocks article title

### ISSUE THREAD

**Issue by littlemiddlebiddle** [steemit/condenser#3385]
**Specifications:
* FF 60.6.1 ESR
* Windows 7
* 1920px window.innerWidth;
* 960px window.innerHeight

**Steps to reproduce the problem:**

1. Navigate to https://steemit.com/steamit/@noisy/steem-bug-bounty-program-how-to-reports-bugs-and-new-ideas in browser and wait for the ad to load

**Actual behavior:**
* Ad blocks part of title
![image](https://user-images.githubusercontent.com/14179790/57563850-10b05100-7371-11e9-905b-b76b84df4ba5.png)

**Expected behavior:**
* Ad does not block title or page content and is as non-invasive as possible

**Comment by quochuy** [comment]
This is a duplicate of https://github.com/steemit/condenser/issues/3281

**Comment by roadscape** [comment]
Blocked by https://github.com/steemit/condenser/pull/3390

**Comment by roadscape** [comment]
https://github.com/steemit/condenser/pull/3404 contains more mitigations

**Comment by JonathanPorta** [comment]
To get ads to show up locally set these environment variables:
```
GPT_ENABLED=true
GOOGLE_AD_CLIENT=ca-pub-9368037717385698
GOOGLE_AD_ENABLED=true 
```

**Comment by quochuy** [comment]
@roadscape I guess this one is already solved

**Comment by roadscape** [comment]
We've experienced some rare but lingering issues, thinking it may be due to CSP violations not being handled.

**Comment by quochuy** [comment]
@roadscape why not move the advert banner outside of the header into the top of the article, above the title. So that the header height is never modified.
